apps/dapp/cypress/e2e/auction-list-page.cy.ts

```diff
@@ -0,0 +1,31 @@
+import { stubGetAuctionByTokenAddressQuery } from "../../src/mocks/stubs/get-auction-by-token-address-query";
+import { AUCTION_TOKEN_ADDRESS } from "../../../../app-config";
+import { COMPONENTS } from "../constants";
+
+const BASE_URL = "http://localhost:5173";
+
+const [firstAuction] = stubGetAuctionByTokenAddressQuery({
+  baseTokenAddress: AUCTION_TOKEN_ADDRESS,
+}).batchAuctionLots;
+
+describe("Auction List Page", () => {
+  it("Should load the root page", () => {
+    cy.visit(BASE_URL);
```

Comment:
This will pass even if the home page fails and an error page is shown.
I think we can omit this top level test from here as its covered in the smoke tests:

`smoke.cy.ts`
```
it("renders home page", () => {
    cy.visit(URLS.HOME);
    cy.shouldRenderPageWithId("__AXIS_HOME_PAGE__");
  });
```